recipes/pycoal/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pycoal" %}
+{% set version = "0.5.1" %}
+{% set sha256 = "d066af5e92d1d8250d972c7a683f70403df40e2aa3bac2d45655b301df7c7a18" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - numpy
+    - spectral
+
+test:
+  imports:
+    - pycoal
+
+about:
+  home: https://github.com/capstone-coal/pycoal/
+  license: Apache 2.0
+  license_family: Apache
+  license_file: LICENSE
```

Comment:
@tboggs The licensing implications of importing a Python module are not as clear as linking a C library, so a case could be made that the application program is simply using software installed on the system rather than being a derived work. On the other hand, we used `spectral` for much of the framework of this version of `pycoal` so that argument might not hold up. GPL's strong terms encourage developers to err on the side of caution, which is why we will relicense `pycoal` under GPLv2.

The choice of license is a personal, practical, and political decision. [GPLv3 adds stronger protections](https://www.gnu.org/licenses/quick-guide-gplv3.html) for certain corner cases which are designed to promote free software but are not without critics. An alternative to relicensing `spectral` under GPLv3 would be to add an "any later version" clause, however this requires you to trust that the FSF will always release an acceptable license.

No action is probably the easiest thing to do right now since everything has worked out for us. Thanks!